bioblend/galaxy/tools/__init__.py

```diff
@@ -138,6 +138,24 @@ def upload_file(self, path, history_id, **keywords):
         finally:
             payload["files_0|file_data"].close()
 
+    def upload_from_ftp(self, history_id, path, **keywords):
```

Comment:
I'd swap `history_id` and `path` to be coherent with `upload_file()` and `paste_content()` methods.